### PR TITLE
clean up @package and @link in header

### PR DESCRIPTION
**Pull request by teryhill**
clean up @package and @link in header removed some unused code missed in
date picker work #332

### CODE REVIEW

**Comment by tmccormi** [comment]
Well, I'm not going to look at all 177 files.  I assume that nothing in this is high risk?

File: ccr/createCCRImmunization.php [hunk 1]
```diff
@@ -15,72 +15,72 @@
...
  */
 
 
-	$result = getImmunizationData();
-	$row = sqlFetchArray($result);
+    $result = getImmunizationData();
+    $row = sqlFetchArray($result);
 
-	do {
+    do {
```
Comment (aethelwulffe) [inline ccr/createCCRImmunization.php]: newline after beginning a block?  Intentional?

File: composer.json [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
+    "name" : "LibreEHR",
+    "type" : "project",
+    "description" : "LibreHealth EHR - Free Open Source Electronic Health Records",
+    "keywords" : [ "libreehr", "openemr" ],
+    "authors" : [ { 
+        "name" : "Ken Chapple",
+        "email" : "ken@mi-squared.com"
+    } ],
+    "require" : { 
+        "php" : ">=5.3.0"
```
Comment (aethelwulffe) [inline composer.json]: Will our stuff run on 5.3?  I am almost certain that it must be 5.4, and possibly 7.0 specifically.

File: composer.json [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
     },
-    "autoload": {
-        "psr-4": {
-            "PluginSystem\\": "library/pluginsystem/",
-            "Framework\\": "library/vendor/libreehr/Framework/"
+    "autoload" : { 
+        "psr-4" : { 
+            "PluginSystem\\" : "library/pluginsystem/",
```
Comment (aethelwulffe) [inline composer.json]: This changing?

File: controllers/C_Prescription.class.php [hunk 4]
```diff
@@ -651,253 +651,253 @@ function multiprintcss_action($id = "") {
...
+            $this->multiprintcss_postfooter();
 
         }
 
-	function _print_prescription_old($p, & $toFile) {
-		require_once ($GLOBALS['fileroot'] . "/library/classes/class.ezpdf.php");
-		$pdf = new Cezpdf($GLOBALS['rx_paper_size']);
-		$pdf->ezSetMargins($GLOBALS['rx_top_margin']
+    function _print_prescription_old($p, & $toFile) {
+        require_once ($GLOBALS['fileroot'] . "/library/classes/class.ezpdf.php");
```
Comment (aethelwulffe) [inline controllers/C_Prescription.class.php]: This commit will stop on the others that I have had rotting for a while.  I will have to start over, or dump them if we commit this.

Comment (teryhill) [inline controllers/C_Prescription.class.php]: Sorry I will kill this commit

**Comment by aethelwulffe** [comment]
Why did you kill it?

**Comment by teryhill** [comment]
redid it in smaller chunks to allow easier review and cleaner intergration

**Pull request closed without merging**